README.md

```diff
@@ -1,212 +1,57 @@
-
-Once the dashboard `STATUS` is `Running`, the dashboard can be accessed by following instructions in the [Accessing the Dashboard](#accessing-the-dashboard) section. 
-
-Optionally, the dashboard can be used with the Tekton Webhooks Extension (see our [Getting Started](https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md) guide).
-
-### Installing a nightly build
-
-Four nightly builds are available: (plain kube or Openshift) * (read-only or read-write):
-
-To install your preferred flavour use one of these four commands:
-
-```bash
-kubectl apply -f https://storage.googleapis.com/tekton-releases-nightly/dashboard/latest/tekton-dashboard-release.yaml
-kubectl apply -f https://storage.googleapis.com/tekton-releases-nightly/dashboard/latest/tekton-dashboard-release-readonly.yaml
-kubectl apply -f https://storage.googleapis.com/tekton-releases-nightly/dashboard/latest/openshift-tekton-dashboard-release.yaml
-kubectl apply -f https://storage.googleapis.com/tekton-releases-nightly/dashboard/latest/openshift-tekton-dashboard-release-readonly.yaml
-```
-
-### Installing from a development environment
-
-As a developer you can install nightly builds, or a local build. Nightly builds come in the usual four flavours.
-
-```shell
-# Plain Kube
-kustomize build overlays/latest | ko apply -f -
-kustomize build overlays/latest-locked-down | ko apply -f -
-
-# OpenShift
-kustomize build overlays/latest-openshift --load_restrictor=LoadRestrictionsNone \
- | ko resolve -f - | kubectl apply -f - --validate=false
-kustomize build overlays/latest-openshift-locked-down --load_restrictor=LoadRestrictionsNone \
- | ko resolve -f - | kubectl apply -f - --validate=false
-```
-
-In read-only mode, buttons and sections of the Dashboard will not be displayed (for example, you won't have the ability to create, stop, and delete PipelineRuns).
-
-Development installation of the Dashboard uses `ko`:
-
-```bash
-$ docker login
-$ export KO_DOCKER_REPO=docker.io/<mydockername>
-$ ./install-dev.sh
-```
-
-The `install-dev.sh` script will build and push an image of the Tekton Dashboard to the Docker registry which you are logged into. Any Docker registry will do, but in this case it will push to Dockerhub. It will also build the static web content using `npm` scripts.
-
-### Supported backend options
-
-The dashboard backend supports a number of options through command line arguments.
+It allows users to manage and view Tekton resources creation, execution, and completion.
 
-These options are documented below:
+Among other things, the Tekton Dashboard supports:
+- Filtering resources by labels
+- Realtime view of PipelineRuns and TaskRuns logs
+- View resources details and YAML
+- Show resources for the whole cluster or limit visibility to a particular namespace
+- Import resources directly from a git repository
+- Setup webhooks for automatically triggering pipelines on git events (github and gitlab supported)
```

Comment:
that's the webhooks-extension, not the core dashboard

Reply:
fixed